libraries/rust_install.rb

```diff
@@ -25,71 +25,111 @@ class Resource::RustInstall < Resource::LWRPBase
     default_action :install
 
     attribute :version, kind_of: String, name_attribute: true
-    attribute :channel, kind_of: String
+    attribute :channel, kind_of: String, default: 'stable'
+    attribute :prefix, kind_of: String, default: lazy { ChefConfig.windows? ? '/usr/local' : '/usr/local' }
```

Comment:
I think we are talking about two different things. I was referring the default value returned on Windows for the prefix attribute.